### ISSUE TITLE
error in configure.ac logic

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yss163com/libjingle#17]
```
What steps will reproduce the problem?
1. compile & install speex to a non-standard location
2. Run the configure script with the --with-speex option

What is the expected output? What do you see instead?
At the end of the script, it lists the supported codecs. The line for speex
indicates that is NOT supported.

The problem is in the configure.ac file, when it tests for the existence of
the speex includes & libraries. It doesn't set the CPPFLAGS and LDFLAGS at
the right time, before the test.

Attached is my diff for updating the configure.ac file to work correctly. I
basically copied the logic used for testing the iLBC libraries.

If you install speex to the standard location (/usr), then you will not
have this problem, because the standard CPPFLAGS and LDFLAGS will work for you.
```

Original issue reported on code.google.com by `mattwinter` on 16 Nov 2007 at 4:59

Attachments:
- [configure.ac.patch](https://storage.googleapis.com/google-code-attachments/libjingle/issue-17/comment-0/configure.ac.patch)
